src/status_im/contexts/wallet/create_account/select_keypair/view.cljs

```diff
@@ -26,25 +28,52 @@
       :accessibility-label :import-private-key
       :label               (i18n/label :t/import-private-key)}]]])
 
-(def accounts
-  [{:account-props {:customization-color :turquoise
-                    :size                32
-                    :emoji               "\uD83C\uDFB2"
-                    :type                :default
-                    :name                "Trip to Vegas"
-                    :address             "0x0ah...71a"}
-    :networks      [{:network-name :ethereum :short-name "eth"}
-                    {:network-name :optimism :short-name "opt"}]
-    :state         :default
-    :action        :none}])
+(defn parse-accounts
+  [given-accounts]
+  (->> given-accounts
+       (filter (fn [{:keys [path]}]
+                 (not (string/starts-with? path constants/path-eip1581))))
+       (map (fn [{:keys [colorId emoji name address]}]
+              {:account-props {:customization-color (if (not-empty colorId) (keyword colorId) :blue)
+                               :size                32
+                               :emoji               emoji
+                               :type                :default
+                               :name                name
+                               :address             address}
+               :networks      [{:network-name :ethereum :short-name "eth"}
+                               {:network-name :optimism :short-name "opt"}
+                               {:network-name :arbitrum :short-name "arb1"}]
```

Comment:
This is temporary. The account networks are not included in the keypair data returned from status-go, so need to read the account networks in another way. Will be address in that issue: https://github.com/status-im/status-mobile/issues/18993